spec/spec_helper.rb

```diff
@@ -2,6 +2,8 @@
 require 'rspec-puppet-facts'
 include RspecPuppetFacts
 
+$LOAD_PATH.unshift(File.expand_path(File.dirname(__FILE__) + '/../'))
```

Comment:
this will get killed by the next modulesync. This file is a template. Is this actually needed?

Reply:
if i remove that line, than i get an error while running the tests. - maybe you know, how this could be fixed?
---
An error occurred while loading ./spec/unit/openvpn_module_spec.rb.
Failure/Error: require 'lib/facter/openvpn'

LoadError:
  cannot load such file -- lib/facter/openvpn

Reply:
What about moving that $LOAD_PATH thing into spec/unit/openvpn_module_spec.rb? That may work.